app/src/main/java/com/aritra/notify/ui/screens/notes/addEditScreen/AddEditScreen.kt

```diff
@@ -148,6 +161,8 @@ fun AddEditScreen(
     }
     val formattedDateTime = SimpleDateFormat(Const.DATE_TIME_FORMAT, Locale.getDefault()).format(dateTime ?: 0)
     val formattedCharacterCount = "${(title.length) + (description.length)} characters"
+    val formattedWordCount = "${countWords(title) + countWords(description)} words"
+    val formattedReadTime = "${calculateTime((countWords(title) + countWords(description)),wordsPerMinute)} sec (reading time)"
```

Comment:
Make it only `sec read`.
e.g `1 sec read` , `6 sec read`

Reply:
it is already seconds read you can check out video again :) or you are requesting something else

Reply:
Currently, it is `1 sec (reading time)` I was telling you to write it as `1 sec read`

Reply:
gotcha already bro